LayoutTests/http/wpt/service-workers/update-service-worker.https.html

```diff
@@ -10,10 +10,10 @@
 var registration;
 var iframe;
 promise_test(async (test) => {
-    registration = await navigator.serviceWorker.getRegistration("resources");
+    registration = await navigator.serviceWorker.getRegistration("resources/");
```

Comment:
Was the old value simply wrong? Or is there something deeper here like a bug? Why didn’t people working on other platforms discover this mistake?

Reply:
And also, why was the failure flaky?

Reply:
The old value is wrong, as is tested by LayoutTests/imported/w3c/web-platform-tests/service-workers/service-worker/registration-security-error.https.html.
Note that this test is WebKit specific and not a WPT one. Without the fix, the test fails in Chrome and Firefox. With the fix, it success in Chrome and Firefox.

The failure was flaky at some point but the test is now consistently failing since a long time.
It might be the test is still flaky after the fix, we can discover this once we land this correctness fix.
Maybe I should have changed the name of the bug, or made that change on a new bug.

Reply:
The test itself is a private WebKit test (as all tests in LayoutTests/http/wpt).
The official WPT tests are in LayoutTests/imported/w3c/web-platform-tests.
We could contemplate upstreaming some of the tests in LayoutTests/http/wpt to GitHub WPT.
This would for instance mean moving LayoutTests/http/wpt/service-workers/update-service-worker.https.html to LayoutTests/imported/w3c/web-platform-tests/service-workers.